### PR TITLE
Add documentation about timestream schema

### PR DESCRIPTION
**Pull request by devpow112**
Will cross link this with the confluence pages once it's merged.

### CODE REVIEW

File: docs/storage-schema.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Storage Schema
+
+All report data once submitted to the back-end is stored in [AWS Timestream].
+The data is split across 2 tables ([`summary`](#summary) and
+[`details`](#details)) to better store common items, reduce duplication and
+allow for more targeted queries. Below are a breakdown of the various tables and
```
Comment (sarahboyd) [inline docs/storage-schema.md:6]:
```suggestion
allow for more targeted queries. Below is a breakdown of the various tables and
```

Comment (devpow112) [reply docs/storage-schema.md:6]: Grammar be so hard yo!

**Review by sarahboyd — APPROVED** [review]
LGTM

File: docs/storage-schema.md (hunk 1 above)
Comment (sarahboyd) [reply docs/storage-schema.md]: especially on a Monday :P

**Pull request merged**